# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux26sh/serial_sh.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ru linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/dreamcast_defconfig linux-2.6.13.4-sh-patch-483/arch/sh/configs/dreamcast_defconfig
--- linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/dreamcast_defconfig	2005-10-30 11:13:27.806709216 +0100
+++ linux-2.6.13.4-sh-patch-483/arch/sh/configs/dreamcast_defconfig	2005-10-30 11:15:24.018042408 +0100
@@ -555,8 +555,8 @@
 #
 # Non-8250 serial port support
 #
-CONFIG_SERIAL_SH_SCI=y
-CONFIG_SERIAL_SH_SCI_CONSOLE=y
+CONFIG_SERIAL_SH_SCI=n
+CONFIG_SERIAL_SH_SCI_CONSOLE=n
 CONFIG_SERIAL_CORE=y
 CONFIG_SERIAL_CORE_CONSOLE=y
 # CONFIG_SERIAL_JSM is not set
diff -ru linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/hs7751rvoip_defconfig linux-2.6.13.4-sh-patch-483/arch/sh/configs/hs7751rvoip_defconfig
--- linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/hs7751rvoip_defconfig	2005-10-30 11:13:27.783712712 +0100
+++ linux-2.6.13.4-sh-patch-483/arch/sh/configs/hs7751rvoip_defconfig	2005-10-30 11:15:24.075033744 +0100
@@ -565,8 +565,8 @@
 #
 # Non-8250 serial port support
 #
-CONFIG_SERIAL_SH_SCI=y
-CONFIG_SERIAL_SH_SCI_CONSOLE=y
+CONFIG_SERIAL_SH_SCI=n
+CONFIG_SERIAL_SH_SCI_CONSOLE=n
 CONFIG_SERIAL_CORE=y
 CONFIG_SERIAL_CORE_CONSOLE=y
 # CONFIG_SERIAL_JSM is not set
diff -ru linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/landisk_defconfig linux-2.6.13.4-sh-patch-483/arch/sh/configs/landisk_defconfig
--- linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/landisk_defconfig	2005-10-30 11:13:27.750717728 +0100
+++ linux-2.6.13.4-sh-patch-483/arch/sh/configs/landisk_defconfig	2005-10-30 11:15:24.104029336 +0100
@@ -713,8 +713,8 @@
 #
 # Non-8250 serial port support
 #
-CONFIG_SERIAL_SH_SCI=y
-CONFIG_SERIAL_SH_SCI_CONSOLE=y
+CONFIG_SERIAL_SH_SCI=n
+CONFIG_SERIAL_SH_SCI_CONSOLE=n
 CONFIG_SERIAL_CORE=y
 CONFIG_SERIAL_CORE_CONSOLE=y
 # CONFIG_SERIAL_JSM is not set
diff -ru linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/microdev_defconfig linux-2.6.13.4-sh-patch-483/arch/sh/configs/microdev_defconfig
--- linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/microdev_defconfig	2005-10-30 11:13:27.753717272 +0100
+++ linux-2.6.13.4-sh-patch-483/arch/sh/configs/microdev_defconfig	2005-10-30 11:15:24.125026144 +0100
@@ -464,8 +464,8 @@
 #
 # Non-8250 serial port support
 #
-CONFIG_SERIAL_SH_SCI=y
-CONFIG_SERIAL_SH_SCI_CONSOLE=y
+CONFIG_SERIAL_SH_SCI=n
+CONFIG_SERIAL_SH_SCI_CONSOLE=n
 CONFIG_SERIAL_CORE=y
 CONFIG_SERIAL_CORE_CONSOLE=y
 CONFIG_UNIX98_PTYS=y
diff -ru linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/se7300_defconfig linux-2.6.13.4-sh-patch-483/arch/sh/configs/se7300_defconfig
--- linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/se7300_defconfig	2005-10-30 11:13:27.749717880 +0100
+++ linux-2.6.13.4-sh-patch-483/arch/sh/configs/se7300_defconfig	2005-10-30 11:15:24.164020216 +0100
@@ -355,8 +355,8 @@
 #
 # Non-8250 serial port support
 #
-CONFIG_SERIAL_SH_SCI=y
-CONFIG_SERIAL_SH_SCI_CONSOLE=y
+CONFIG_SERIAL_SH_SCI=n
+CONFIG_SERIAL_SH_SCI_CONSOLE=n
 CONFIG_SERIAL_CORE=y
 CONFIG_SERIAL_CORE_CONSOLE=y
 # CONFIG_UNIX98_PTYS is not set
diff -ru linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/se73180_defconfig linux-2.6.13.4-sh-patch-483/arch/sh/configs/se73180_defconfig
--- linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/se73180_defconfig	2005-10-30 11:13:27.752717424 +0100
+++ linux-2.6.13.4-sh-patch-483/arch/sh/configs/se73180_defconfig	2005-10-30 11:15:24.179017936 +0100
@@ -337,8 +337,8 @@
 #
 # Non-8250 serial port support
 #
-CONFIG_SERIAL_SH_SCI=y
-CONFIG_SERIAL_SH_SCI_CONSOLE=y
+CONFIG_SERIAL_SH_SCI=n
+CONFIG_SERIAL_SH_SCI_CONSOLE=n
 CONFIG_SERIAL_CORE=y
 CONFIG_SERIAL_CORE_CONSOLE=y
 # CONFIG_UNIX98_PTYS is not set
diff -ru linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/se7705_defconfig linux-2.6.13.4-sh-patch-483/arch/sh/configs/se7705_defconfig
--- linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/se7705_defconfig	2005-10-30 11:13:27.760716208 +0100
+++ linux-2.6.13.4-sh-patch-483/arch/sh/configs/se7705_defconfig	2005-10-30 11:15:24.200014744 +0100
@@ -539,8 +539,8 @@
 #
 # Non-8250 serial port support
 #
-CONFIG_SERIAL_SH_SCI=y
-CONFIG_SERIAL_SH_SCI_CONSOLE=y
+CONFIG_SERIAL_SH_SCI=n
+CONFIG_SERIAL_SH_SCI_CONSOLE=n
 CONFIG_SERIAL_CORE=y
 CONFIG_SERIAL_CORE_CONSOLE=y
 CONFIG_UNIX98_PTYS=y
diff -ru linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/se7750_defconfig linux-2.6.13.4-sh-patch-483/arch/sh/configs/se7750_defconfig
--- linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/se7750_defconfig	2005-10-30 11:13:27.756716816 +0100
+++ linux-2.6.13.4-sh-patch-483/arch/sh/configs/se7750_defconfig	2005-10-30 11:15:24.221011552 +0100
@@ -517,8 +517,8 @@
 #
 # Non-8250 serial port support
 #
-CONFIG_SERIAL_SH_SCI=y
-CONFIG_SERIAL_SH_SCI_CONSOLE=y
+CONFIG_SERIAL_SH_SCI=n
+CONFIG_SERIAL_SH_SCI_CONSOLE=n
 CONFIG_SERIAL_CORE=y
 CONFIG_SERIAL_CORE_CONSOLE=y
 CONFIG_UNIX98_PTYS=y
diff -ru linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/sh03_defconfig linux-2.6.13.4-sh-patch-483/arch/sh/configs/sh03_defconfig
--- linux-2.6.13.4-sh-patch-483/arch/sh/configs.old/sh03_defconfig	2005-10-30 11:13:27.782712864 +0100
+++ linux-2.6.13.4-sh-patch-483/arch/sh/configs/sh03_defconfig	2005-10-30 11:15:24.266004712 +0100
@@ -635,8 +635,8 @@
 #
 # Non-8250 serial port support
 #
-CONFIG_SERIAL_SH_SCI=y
-CONFIG_SERIAL_SH_SCI_CONSOLE=y
+CONFIG_SERIAL_SH_SCI=n
+CONFIG_SERIAL_SH_SCI_CONSOLE=n
 CONFIG_SERIAL_CORE=y
 CONFIG_SERIAL_CORE_CONSOLE=y
 # CONFIG_SERIAL_JSM is not set
